cmake/oneDPLWindowsIntelLLVMConfig.cmake

```diff
@@ -11,7 +11,17 @@
 # See https://llvm.org/LICENSE.txt for license information.
 #
 ##===----------------------------------------------------------------------===##
-cmake_minimum_required(VERSION 3.15.0)
+
+if(CMAKE_HOST_WIN32)
```

Comment:
I would prefer if there was a space after the `if` to be consistent with the other `if` statements.

Reply:
done, thanks.